tckprocess.adoc

```diff
@@ -0,0 +1,406 @@
+going to market; Challenges SHOULD be given a high priority by the
+specification contributors and resolved in a timely manner. Two weeks or less
+SHOULD be considered the ideal period of time to resolve a challenge.
+Challenges may go longer as needed, but as a rule SHOULD avoid months.
+
+If consensus cannot be reached by the specification contributors for a
+prolonged period of time, the default recommendation is to exclude the
+tests and address the dispute in a future revision of the specification.
+
+[#_accepted_challenges]
+#### Accepted Challenges
+
+A consensus that a test produces invalid results will result in the
+exclusion of that test from certification requirements, and an immediate
+update and release of an official distribution of the TCK including the
+new exclude list. The associated `challenge` issue MUST be closed with
+an `accepted` label to indicate it has been resolved.
+
+[#_rejected_challenges_and_remedy]
+#### Rejected Challenges and Remedy
+
+When a `challenge` issue is rejected, it MUST be closed with a label of
+`invalid` to indicate it has been rejected. The appeal process for
+challenges rejected on technical terms is outlined in Escalation Appeal.
+If, however, an implementer feels the TCK challenge process was not
+followed, an appeal issue MUST be filed with the specification's
+issue tracker using the label `challenge-appeal`. A contributor MUST
+escalate the issue with the MicroProfile Specification Committee via email
+(<microprofile-wg@eclipse.org>). The committee will evaluate the matter
+purely in terms of due process. If the appeal is accepted, the original
+TCK challenge issue will be reopened and a label of `appealed-challenge`
+added, along with a discussion of the appeal decision, and the
+`challenge-appeal` issue with be closed. If the appeal is rejected, the
+`challenge-appeal` issue MUST be closed with a label of `invalid`.
+
+image:tckprocess.png[TCK Process]
+
+[#_excludes]
+## Excludes
+
+Excludes MUST be included in the specification's TCK release in a format that is
+compatible with the testing framework in use so that as the excludes are
+updated, the affected tests are automatically removed from the test
+suite.
+
+[#_improvement]
+## Improvement
+
+Requests for improvement to tests MUST simply be created as issues with
+a label of `enhancement` in the specification's TCK issue
+tracker.
+
+[#_certification_of_compatibility]
+## Certification of Compatibility
+
+MicroProfile is a self-certification ecosystem. If you wish to have your
+implementation listed on the official https://microprofile.io[microprofile.io]
+implementations page for the given specification, a certification
+request as defined in this section is required.
+
+
+[#_filing_a_certification_request]
+### Filing a Certification Request
+
+Requests to be acknowledged as a certified implementation MUST be filed
+under the github repo https://github.com/eclipse/microprofile via the specification's issue tracker using the label
```

Comment:
@Emily-Jiang this seems a little contradicting to the previous chapter which seems to say that the implementation can certify against individual specifications. https://github.com/eclipse/microprofile is used for the whole MP parent umbrella (currently 8 specs). So if the implementation wants to certify only one MP specification, will that be done as an issue in that specification repo (like we now do CCR) or in https://github.com/eclipse/microprofile? There is already some confusion about this. Is there also a possibility to list implementation used for CCR as automatically compatible so we don't need to repeat the certification issue twice?

Reply:
yes, my intent is to cerify the umbrella release only, which automatically certifies all of the enclosed specs without the need to repeat 8 CCRs. I will clarify this further.

Reply:
I have clarified this in the new commit.